### DISCUSSION TITLE
Should be able to tap and hold to copy link to search result

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#53972] [Mobile] [answered]
GitHub Mobile beta

1. Search in discussions
2. Tap and hold on a search result

![Screenshot_20230428-093404.png](https://user-images.githubusercontent.com/2119212/235162961-ec54c4bc-f4ee-4372-9154-65ec71b5021f.png)

Apparently GitHub Mobile sometimes supports tap and hold, but, not here. I'd expect to be able to copy a link/share a link to the result.

**Comment by taki-on** [comment, marked as answer]
Thank you for your suggestion. We will have 👀 on it and see what we can do to improve the search UX.
